methods.tex

```diff
@@ -401,12 +401,12 @@ \subsection*{Random number generation}
 from a system-dependent provider, or with a user-provided seed. The seed
 sequence then transforms the initial set of entropy into a sequence of
 high-quality pseudorandom integers, which can be used to initialize multiple bit
-generators deterministically. A key design goal of a seed sequence was to be
-splittable in the sense that a seed sequence can be used to produce child
-sequences that are distinct from their parents or other ancestors. This
-capability allows a seed sequence to be used in large distributed applications
+generators deterministically. A key design element of a seed sequence is that
+it can be used to spawn child \code{SeedSequence}s which are distinct from
```

Comment:
What does distinct mean here?  Do you think we need to be more precise, or is that enough?

Reply:
Good one, just realized, we also really need to cite Professor O'Neils work here if we did not already. She probably has some nice phrase for distinct, although the opposite direction to just tell "its safe to use it to create as many generators as you wish" may be just as well.